rpc/rpc-wrappers.go

```diff
@@ -955,6 +956,26 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 		return
 	})
 
+	rpc.Handle(MethodRevokeAuthorizationsByDomain, func(req []byte) (response []byte, err error) {
+		var ident core.AcmeIdentifier
+		err = json.Unmarshal(req, &ident)
+		if err != nil {
+			return
+		}
+		aRevoked, paRevoked, err := impl.RevokeAuthorizationsByDomain(ident)
+		if err != nil {
+			return
+		}
+		var raResp struct {
```

Comment:
This struct, and the one for the arguments, should be defined once in this file and used by both sides of the RPC, to ensure it doesn't drift out of sync.